pkg/vm/engine/tae/containers/types.go

```diff
@@ -35,6 +35,11 @@ var DefaultAllocator = stl.DefaultAllocator
 // var DefaultAllocator = alloc.NewAllocator(int(common.G) * 100)
 
 var NewBytes = stl.NewBytes
```

Comment:
I really don't understand why you want to alias this.    I know this is minor, but why?   Just to save a few key strokes?

Reply:
Removed